### DISCUSSION TITLE
Missing/deleted page app.diagrams.net

### DISCUSSION THREAD

**Discussion by agungrizkiono** [jgraph/drawio#4760] [General]
I have many architecture at draw.io and now my page is dissapear/deleted. I don't know why it happened like that.
I had to do backup my page but at 2022. Now I need my page back. What I must to do?

That's my missing page: https://app.diagrams.net/#G1we0DI6GNDHVZR5sL2DhkcoCKP0cElolx#%7B%22pageId%22%3A%22Yyh3MH3WbpL16vV3iTfy%22%7D

<img width="576" alt="page draw io" src="https://github.com/user-attachments/assets/d1273a08-5327-49ce-82d5-7df64f507429">

**Comment by davidjgraph** [comment]
The file is stored in google drive. You need to find the file in your google drive, or work out if it's in someone's elses google drive (or a different account of yours).
